doc/whats_new/_contributors.rst

```diff
@@ -169,3 +169,5 @@
 .. _Roman Yurchak: https://github.com/rth
 
 .. _Hanmin Qin: https://github.com/qinhanmin2014
+
+.. _Adrin Jalali: https://github.com/adrinjalali
```

Comment:
We should have a separate PR adding our names to this list. 😅

Reply:
lol, yeah, I've added mine in a few PRs, it'll be there once one of them gets merged lol